Mêlée creatures no longer fumble around when in combat without a mêlé…

AdamPlenty opened the pull request:
…e attack

Fixes #3164

Loobinex commented:
@AdamPlenty what did you test it with and what was your expectation?

I modified the skillset of a lvl10 spider to be this:
```
[experience]
Powers = NULL SLOW NULL FREEZE NULL NULL NULL NULL NULL NULL
```
and it would only flee, and never fight and use his slow or freeze attacks.

AdamPlenty commented:
> @AdamPlenty what did you test it with and what was your expectation?
> 
> I modified the skillset of a lvl10 spider to be this:
> 
> ```
> [experience]
> Powers = NULL SLOW NULL FREEZE NULL NULL NULL NULL NULL NULL
> ```
> 
> and it would only flee, and never fight and use his slow or freeze attacks.

What was your Spider against? Would it have fled anyway? I modified a Knight to have only Arrow, and everything worked as expected. I removed Arrow (so he had no attacks), and he just wandered around (not fumbling, though), as expected. Gave him Hand-to-Hand back, and he used it as expected.

Loobinex commented:
It was a lvl10 spider versus a lvl1 dwarf. It should not flee.

AdamPlenty commented:
Works fine for me 😕.
